app/services/s3_query_service.rb

```diff
@@ -85,6 +85,40 @@ def client
     @client ||= Aws::S3::Client.new(region: region, credentials: credentials)
   end
 
+  def model
+    @model ||= Work.find_by(doi: @doi)
+  end
+
+  def model_s3_files
```

Comment:
@jrgriffiniii -- If I understand this correctly `model_s3_files` includes files the user added via PDC Describe upload feature and that PDC Described pushed to S3, whereas `client_s3_files` includes files all files in the S3 folder (regardless of how they got there, e.g. uploaded via PDC Describe or manually uploaded to S3). 

`s3_files` returns the combination of these two methods and handles duplicates gracefully. Is that understanding correct? If so, could we add a comment to these methods to clarify this?